src/test/java/com/exactpro/th2/common/event/bean/TreeTableTest.java

```diff
@@ -40,8 +40,10 @@ public void testSerializationRow() throws IOException {
 
         TreeTableBuilder treeTableBuilder = new TreeTableBuilder();
         TreeTable treeTable = treeTableBuilder.row("FirstRow", row).build();
-        com.exactpro.th2.common.grpc.Event event =
-                Event.start().bodyData(treeTable).bookName(BOOK_NAME).toProto(toEventID(BOOK_NAME, "id"));
+        com.exactpro.th2.common.grpc.Event event = Event
+                .start()
+                .bodyData(treeTable)
+                .toProto(toEventID(BOOK_NAME, "id"));
```

Comment:
Maybe extract the `toEventID(BOOK_NAME, "id")` to a constant value

Reply:
done